### DISCUSSION TITLE
[ERROR] Unknown migration status, aborting database update

### DISCUSSION THREAD

**Discussion by drboone** [kimai/kimai#3953] [On-Premise: Installation & Updates]
I'm working on building a new Kimai installation to replace our unforgiveably old setup.  This old setup was built in 2016.  The configuration table contains:
```
| revision | 1389  |
| version | 1.1.0 |
```

I've done the following:

1. Created a new vhost with the necessary web server (nginx 1.22), composer (2.2.18) and PHP (8.2) parts
2. Set up a new mysql (5.7) database, grants, etc
3. Checked out the git repo, using the 1.x branch
4. Run `composer install --optimize-autoloader -n`
5. Placed a .env file with appropriate customizations
6. Loaded a dump of the old database into the new one
7. Renamed all the tables from `kimai_*` to `kimai2_*` based on a note in https://github.com/kimai/kimai/blob/main/UPGRADING.md
8. Run `bin/console kimai:update`

The subject message appears, and console exits with rc=8.

So obviously I'm doing something stoopid.  Someone whack me with a clue stick please?  Thanks!

**Comment by kevinpapst** [comment]
Here is your stick: https://www.kimai.org/documentation/migration-v1.html 😁

**Comment by drboone** [comment]
Even though I'm building on the 1.x branch?

**Reply by kevinpapst** [reply]
No, I am building this software for more than 10 years and have no clue what I am talking about 🤷 
You are using https://github.com/kimai/kimai1/ and want to switch to completely rewritten Kimai (2).

**Comment by drboone** [comment]
For future reference of others who may arrive here, the confusion is that there is a Kimai v0.x / v1.x which is "Kimai 1" and lives in the kimai1 repository; and there's a Kimai v1.x / v2.x which is "Kimai 2" and lives in the kimai repository, and it's not readily obvious that there are two kinds of numbers here, or that there's another older repo.
